test/integers.test.ts

```diff
@@ -65,12 +75,4 @@ describe('Integer tests', () => {
       .check()
     ).to.deep.include({ satisfiable: true, example: { a: 90002 } }) */
   })
-
-  it('finds that adding 1000 makes any number larger and shrinks the example', () => {
-    expect(fc.scenario()
-      .exists('a', fc.integer())
-      .then(({a}) => a + 1000 > a)
-      .check()
-    ).to.deep.include({satisfiable: true, example: {a: 0}})
-  })
```

Comment:
Any particular reason why this test is commented out?

Reply:
It is not commented out to me

Reply:
Sorry, I meant deleted xD

Reply:
It was also not deleted xD

Reply:
Nevermind, it moved place... stupid diff...